runtime/gc_base/IndexableObjectAllocationModel.cpp

```diff
@@ -122,6 +141,12 @@ MM_IndexableObjectAllocationModel::initializeIndexableObject(MM_EnvironmentBase
 	GC_ArrayObjectModel *indexableObjectModel = &extensions->indexableObjectModel;
 	J9IndexableObject *spine = (J9IndexableObject*)initializeJavaObject(env, allocatedBytes);
 	_allocateDescription.setSpine(spine);
+	bool isArrayletDataAdjacentToHeader = false;
+	bool isAllIndexableDataContiguousEnabled = extensions->indexableObjectModel.isVirtualLargeObjectHeapEnabled();
```

Comment:
indexableObjectModel is defined already,  see definition above
```suggestion
	bool isAllIndexableDataContiguousEnabled = indexableObjectModel->isVirtualLargeObjectHeapEnabled();
```